src/tests/unit/test-data/foundry-list/foundry-list.expected

```diff
@@ -26,6 +26,48 @@ APRProof: test%AssertTest.setUp():0
     execution time: 0h 0m 0s
 Subproofs: 0
 
+APRProof: test%AssertTest.setUp():1
+    status: ProofStatus.PASSED
+    admitted: False
+    nodes: 3
+    pending: 0
+    failing: 0
+    vacuous: 0
+    stuck: 0
+    terminal: 2
+    refuted: 0
+    bounded: 0
+    execution time: 0h 0m 0s
+Subproofs: 0
+
+APRProof: test%AssertTest.setUp():0
+    status: ProofStatus.PASSED
+    admitted: False
+    nodes: 3
+    pending: 0
+    failing: 0
+    vacuous: 0
+    stuck: 0
+    terminal: 2
+    refuted: 0
+    bounded: 0
+    execution time: 0h 0m 0s
+Subproofs: 0
+
+APRProof: test%AssertTest.setUp():1
+    status: ProofStatus.PASSED
+    admitted: False
+    nodes: 3
+    pending: 0
+    failing: 0
+    vacuous: 0
+    stuck: 0
+    terminal: 2
+    refuted: 0
+    bounded: 0
+    execution time: 0h 0m 0s
+Subproofs: 0
+
```

Comment:
@tothtamas28 I duplicated a proof to test the `get_test_id` function and now it seems the `foundry-list.expected` file has duplicated both proofs.